dbee/models/layout.go

```diff
@@ -36,6 +37,8 @@ func (s LayoutType) String() string {
 		return "history"
 	case LayoutTypeDatabaseSwitch:
 		return "database_switch"
+	case LayoutTypeView:
```

Comment:
I was thinking about that a while ago, and I couldn't make up my mind.
It looks like we have views as a separate layout type now :D